modules/client/src/client-common/modules/graphql.ts

```diff
@@ -4,40 +4,55 @@ import { Context } from "../../client-common/context";
 import { QueryStatus } from "../graphql-queries";
 import { IClientGraphQLCore } from "../interfaces/core";
 
-export class GraphqlModule implements IClientGraphQLCore {
-  private _graphql: GraphQLClient[] = [];
-  private _graphqlIdx: number = -1;
+const GraphqlMap = new Map<GraphqlModule, GraphQLClient[]>();
+const GraphqlIdxMap = new Map<GraphqlModule, number>();
```

Comment:
I would name them in lower case, potentially even with an underscore. Otherwise, it seems like a Class